docs/src/SUMMARY.md

```diff
@@ -1,62 +1,78 @@
-  - [Connect my device to AWS IoT](./tutorials/connect-aws.md)
-  - [Send measurements](./tutorials/send-thin-edge-data.md)
-  - [Raise alarms](./tutorials/raise-alarm.md)
-  - [Send events](./tutorials/send-events.md)
-  - [Monitor my device](./tutorials/device-monitoring.md)
-  - [Manage my device software](./tutorials/software-management.md)
-  - [Configuration management on child-devices](./tutorials/child-device-config-management.md)
-  - [Supported Operations Management for Cumulocity IoT](./tutorials/supported_operations.md)
-  - [Write my software management plugin](./tutorials/write-my-software-management-plugin.md)
-  - [Build Thin Edge for a Yocto Linux distribution](tutorials/yocto-linux.md)
+- [Operate devices](operate/index.md)
+  - [Installation](operate/installation/index.md)
+    - [How to install thin-edge.io?](operate/installation/002_installation.md)
+    - [How to install thin-edge.io on any Linux OS (no deb support)?](operate/installation/015_installation_without_deb_support.md)
+    - [How to install thin-edge manually with openrc](operate/installation/026_how_to_install_thin_edge_manually.md)
+    - [How to install and enable software management?](operate/installation/012_install_and_enable_software_management.md)
+  - [Configuration](operate/configuration/index.md)
+    - [How to configure thin-edge.io?](operate/configuration/006_config.md)
+    - [How to configure the local mqtt bind address and port?](operate/configuration/008_config_local_mqtt_bind_address_and_port.md)
+    - [How to use thin-edge.io with your preferred init system](operate/configuration/019_how_to_use_preferred_init_system.md)
+    - [How to change temp path?](operate/configuration/018_update_config_paths.md)
+    - [Configuration management on child-devices](operate/configuration/child-device-config-management.md)
+  - [Security and Access Control](operate/security/index.md)
+      - [How to create a test certificate?](operate/security/003_registration.md)
+      - [How to add self-signed certificate root to trusted certificates list?](operate/security/010_add_self_signed_trusted.md)
+      - [How to set up client/server authentication for the local MQTT broker](operate/security/029_mqtt_local_broker_authentication.md)
+  - [Connection](operate/connection/index.md)
+    - [How to connect /index cloud end-point?](operate/connection/004_connect.md)
+  - [Processing Telemetry Data](operate/telemetry/index.md)
+    - [How to use `tedge mqtt` module?](operate/telemetry/005_pub_sub.md)
+  - [Operations](operate/operations/index.md)
+    - [How to restart your thin-edge.io device](operate/operations/016_restart_device_operation.md)
+  - [Monitoring](operate/monitoring/index.md)
+    - [How to enable watchdog using systemd?](operate/monitoring/021_enable_tedge_watchdog_using_systemd.md)
+    - [How to connect an external device?](operate/monitoring/013_connect_external_device.md)
+  - [Troubleshooting](operate/troubleshooting/index.md)
```

Comment:
Not sure if a dedicated `Troubleshooting` section would be good idea in the long run as there could be far too many components to troubleshoot in future. Isn't it better to have the docs describing the feature itself to have a troubleshooting section as well at the end of it?

Reply:
The idea is coming from you ;-) https://github.com/thin-edge/thin-edge.io/issues/1048

Reply:
I'm still all-in for troubleshooting guides. Just having second thoughts on whether separate troubleshooting guides for each aspect.